.gitignore

```diff
@@ -38,4 +38,5 @@ public
 # Local resources
 tmp
 cert.pem
-key.pem
\ No newline at end of file
+key.pem
```

Comment:
Do we need these .pem files? I don't think so.

Reply:
I don't think so either--I never had them.  So let's leave them in the .gitignore?